assets/css/main.css

```diff
@@ -1,624 +1,693 @@
+  font-family: 	"mr-eaves-xl-modern";
+  font-weight:700;
+}
+h3{
+  font-size:48px;
+  font-family: 	"mr-eaves-xl-modern";
+  font-weight:700;
+}
+h4{
+  font-size:24px;
+  font-family: 	"mr-eaves-xl-modern";
+  font-weight:700;
+}
+
+p{
+  font-size:18px;
+}
+
+/*Top Bar*/
+
+.top-bar{
+  padding:20px 0 20px 0;margin:0;
+  width:100%;
+  display:none;
+  position:fixed;
+  top:0;left:0;
+  z-index:998;
+  background:rgba(24,34,41, 1);
+  border-bottom:1px solid rgba(47,62,72, 1);
+}
+.top-bar.after-fold{
+  display:block;
+  opacity:1;
+}
+
+.top-bar .inner-wrapper{
+  max-width:1200px;
+  display:block;
+  margin:0 auto;
+  padding:0;
+}
+.top-bar .inner-wrapper img.top-mark{
+  text-align:left;
+  color:#fff1ef;
+  margin:0;padding:0;
+  font-size:24px;
+  float:left;
+  width:30px;
+  margin-right:15px;
+  margin-top:-3px;
+  display:inline-block;
+  position:relative:
+}
+.top-bar .inner-wrapper p.logotype{
+  font-family: 	"mr-eaves-xl-modern";
+  font-weight:400;
+  letter-spacing: 0.1em;
+  text-transform:uppercase;
+  text-align:left;
+  color:#fff1ef;
+  margin:0;padding:0;
+  position:relative;
+  top:4px;
+  float:left;
+}
+.top-bar .inner-wrapper p.cta{
+  text-align:right;
+  color:#fff1ef;
+  margin:5px 0 0 0;padding:0;
+  font-size:16px;
+  float:right;
+  font-family: 	"mr-eaves-xl-modern";
+  font-weight:400;;
+}
+.top-bar .inner-wrapper p.cta a.btn-hire-us{
+  font-size:14px;
+  padding:6px 25px;
+  background:rgba(204, 40, 41, 1).
+  border:none;
+  margin-left:10px;
+}
+/*BILLBOARD*/
 
-#wrapper {
-  height: 100%;
+.splash-image {
+  color: #fff1ef;
+  font-size: 1.35rem;
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -webkit-align-items: center;
+  -ms-flex-align: center;
+  align-items: center;
+  background: url("../img/desk.jpg") no-repeat center/cover;
+  background-size: cover;
+  height:100vh;
+  padding: 0;
+  magin:0;
+  position: relative;
+  border-top:5px solid  #52868e;
+}
+
+.big-splash div.vert-wrap{
+  width:100%;
+  display:block;
+}
+
+.mark{
+  width:70px;
+  margin:0 0 5px -12px;
+
+}
+.big-splash div h1{
+  display:inline-block;
+  width:100%;
+  color: #fff1efFF9;
+  line-height: 96%;
+  margin:0;padding:0;
+  font-size: 120px;
+  color: #FFFFFF;
+  font-family: 	"mr-eaves-xl-modern", Helvetica, sans-serif;
+  font-weight:700;
+}
+
+.big-splash div h2{
+  /* UI Design, Engineeri: */
+  font-family: 	"mr-eaves-xl-modern";
+  font-weight:400;
+  font-size: 18px;
+  color: #fff1efFFF;
+  letter-spacing: 0px;
+  line-height: 100%;
+  text-transform:uppercase;
+  letter-spacing:.1em;
+  margin: 25px 0 50px 0;
+}
+
+.btn-hire-us{
+  color:#fff1ef;
+  padding:12px 40px;
+  font-size:18px;
+  text-transform:uppercase;
+  text-decoration:none;
+  letter-spacing:.05em;
+  background:  rgba(204, 40, 41, 1);
+  border-radius:25px;
+}
+.btn-hire-us:hover{
+  cursor:pointer;
+  opacity:.95
+}
+
+.big-splash div a.btn-more{
```

Comment:
I'd def remove the `div` here and change to a class. The instant someone swaps the element for something else itll totally break.